app/jobs/stats_graphs_job.rb

```diff
@@ -0,0 +1,11 @@
+class StatsGraphsJob < ApplicationJob
+  queue_as :default
+
+  def perform(*args)
+    Stats.fill_users_graph_cache
```

Comment:
I like that you do these in series so that we are not taxing the database with 6 big aggregation queries at the same time.